src/types/oec/okex.ts

```diff
@@ -0,0 +1,294 @@
+import { CryptoNodeData, VersionDockerImage } from '../../interfaces/crypto-node';
+import { defaultDockerNetwork, NetworkType, NodeClient, NodeType, Role, Status } from '../../constants';
+import { Docker } from '../../util/docker';
+import { ChildProcess } from 'child_process';
+import { v4 as uuid } from 'uuid';
+import request from 'superagent';
+import fs from 'fs-extra';
+import path from 'path';
+import os from 'os';
+import { Ethereum } from "../ethereum/ethereum"
+import { filterVersionsByNetworkType } from '../../util';
+
+const coreConfig = `
```

Comment:
Please add the data and keys directories explicitly to the config. We do not want to have to trust the image maintainers to not change file system locations. We need to be able to be explicit about where these are located.

Reply:
I've updated the config with the explicit directories for data and key directories